scipy/stats/morestats.py

```diff
@@ -1560,7 +1572,19 @@ def bartlett(*args):
     .. [2]  Snedecor, George W. and Cochran, William G. (1989), Statistical
               Methods, Eighth Edition, Iowa State University Press.
 
+    .. [3] Park, C. and Lindsay, B. G. (1999). Robust Scale Estimation and
+           Hypothesis Testing based on Quadratic Inference Function. Technical
+           Report #99-03, Center for Likelihood Studies, Pennsylvania State
+           University.
```

Comment:
This only has one citation. On the issue for ``bartlett`` I had suggested to cite the original paper: Bartlett, "Properties of sufficiency and statistical tests"